modules/apps/change-tracking/change-tracking-rest/src/main/java/com/liferay/change/tracking/rest/internal/resource/CTCollectionResource.java

```diff
@@ -216,6 +216,30 @@ else if (_TYPE_ALL.equals(type)) {
 		);
 	}
 
+	@GET
+	@Path("/recent")
+	@Produces(MediaType.APPLICATION_JSON)
+	public List<CTCollectionModel> getRecentCTCollectionModels(
```

Comment:
@zoltancsaszi the whole functionality of this method is already available in the ```public List<CTCollectionModel> getCTCollectionModels(
			@QueryParam("companyId") long companyId,
			@QueryParam("userId") long userId,
			@DefaultValue(_TYPE_ALL) @QueryParam("type") String type,
			@QueryParam("limit") int limit, @QueryParam("sort") String sort)
		throws CTJaxRsException``` method. This is already capable of doing everything this method is aiming for, it seems like the whole pull request is unnecessary.

Reply:
@matethurzo getCTCollectionModels method will return the productionCTCollection as well so we can't use this rest call without modification.